mecanum_drive_controller/test/diff_drive_test.cpp

```diff
@@ -0,0 +1,151 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
Indeed, I have not been able to simulate mecanum wheels in gazebo so I now use VRep. I am not sure how to handle this one... This is the reason why there is currently no unit test for the controller.
This is a blocking problem for me.